src/pybind/mgr/telemetry/module.py

```diff
@@ -0,0 +1,334 @@
+"""
+Telemetry module for ceph-mgr
+
+Collect statistics from Ceph cluster and send this back to the Ceph project
+when user has opted-in
+"""
+import errno
+import json
+import re
+import uuid
+import time
+from datetime import datetime
+from threading import Event
+from collections import defaultdict
+
+from mgr_module import MgrModule
+
+try:
+    import requests
```

Comment:
ceph-common depends on python-requests, so i guess we can assume that `requests` is always available?